app/src/main/java/com/android/solvit/seeker/ui/profile/SeekerProfile.kt

```diff
@@ -276,14 +54,13 @@ fun SeekerProfileScreen(viewModel: SeekerProfileViewModel, navigationActions: Na
   // Display the profile information if it's available
   userProfile.let { profile ->
     Scaffold(
+        modifier = Modifier.testTag("profileScreen"),
         topBar = {
           TopAppBar(
-              modifier = Modifier.testTag("ProfileTopBar"),
               backgroundColor = Color.Blue, // Set blue background
               title = { Text("Profile", color = Color.White) },
               navigationIcon = {
-                // When you go back from profile you diretctly return to home page
-                IconButton(onClick = { navigationActions.navigateTo(Route.SERVICES) }) {
+                IconButton(onClick = { navigationActions.goBack() }) {
```

Comment:
I think that we don't to do a goBack but more a navigation to the HomePage for a smoother user experience